lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,6 +27,24 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instance(instance_name)
```

Comment:
Sorry but I'm confused :/ The class is `InstanceGroup`, no?
In this case "add_instance" is called because instance is being added to an instance group, this is not a collection, but rather GCE's own groups: https://cloud.google.com/compute/docs/reference/latest/instanceGroups/addInstances

Or do you mean extending `Instance` somehow?